microsoft-edge/toc.yml

```diff
@@ -307,7 +307,7 @@
                   href: devtools-guide-chromium/inspect-styles/animations.md
                   displayName: Animations tool
 # Application tool ------------------------------------------------------------
-            - name: Application tool, for storage
+            - name: Application tool, for storage and PWAs
```

Comment:
Note that I changed the title of the article itself, so we should also change this TOC entry.
The only reason that I did not was that I wasn't sure which one was better. Your original title, or the new one I used for the article. Let me know what you think, and feel free to change them all to what you prefer. The important thing is that they're all the same (TOC entry, article H1 title, and frontmatter title entry).

Reply:
You changed the title of the article from what to what?